README.md

```diff
@@ -5,6 +5,18 @@
 [![License](https://img.shields.io/cocoapods/l/AnchorChain.svg?style=flat)](https://cocoapods.org/pods/AnchorChain)
 [![Platform](https://img.shields.io/cocoapods/p/AnchorChain.svg?style=flat)](https://cocoapods.org/pods/AnchorChain)
 
+A wrapper around the NSLayoutAnchor system, driven by the following set of design goals / trade-offs:
+
+- Ease of expression should be proportional to frequency of use. In other words: constraining a view to its superview's edges should be trivial, and indeed it is: **view.anchor()**
+
+- Intuitiveness at the point of use. Constraining a view´s top to the bottom of another view should probably be expressed as: **view.anchor(.top, to: .bottom, of: anotherView)**, and indeed it is.
+- Trade-off: autocompletion will be of limited use, as the anchor method is heavily overloaded.
+
+- Chainability to promote a little more declarative layout code. Adding some color to a view should be as easy as **UIView().anchored(to: view).backgroundColor = .blue**, and indeed it is.
+
+- Be a little pro-active with the view hierarchy. When constraining anchors to match another view, it could be assumed that the receiver, if it is without superview, should be added as subview to the other view, and indeed it is.
```

Comment:
```suggestion
- Be a little pro-active with the view hierarchy. When constraining anchors to match another view, such as this: **UIActivityIndicatorView().anchor(.centerX, .centerY, to: view)**, it could be assumed that the receiver, if it is without superview, should be added as subview to the other view, and indeed it is.
```